devices/ruijia.js

```diff
@@ -0,0 +1,101 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const extend = require('zigbee-herdsman-converters/lib/extend');
+const globalStore = require('zigbee-herdsman-converters/lib/store');//这里原本没有，在最新版本的fromzigbee.js里复制过来的，路径要和上面的其他一样，不能用...代替
+const e = exposes.presets;
+const ea = exposes.access;
+
+const occupancyTimeout = 10; // In seconds
+
+const getKey = (object, value) => {
+    for (const key in object) {
+        if (object[key] == value) return key;
+    }
+};
+const bind = async (endpoint, target, clusters) => {
+    for (const cluster of clusters) {
+        await endpoint.bind(cluster, target);
+    }
+};
+
+const fzLocal = {
+	REJIA_3keys_armmode: {
+        cluster: 'ssIasAce',
+        type: 'commandArm',
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.armmode != null) {
+                const lookup = {
+                    0: 'disarm',
+                    2: 'sos',
+                    3: 'arm',
+                };
+                const value = msg.data.armmode;
+                return {action: lookup[value] || `armmode_${value}`};
+            }
+        },
+    },
+	
+
+    REJIA_vibration_alarm_1_with_timeout: {
```

Comment:
Looks the same as `ias_vibration_alarm_1_with_timeout`, please use that instead.